core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/WindowGroupLimitParser.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.tool.planparser
+
+import com.nvidia.spark.rapids.tool.qualification.PluginTypeChecker
+
+import org.apache.spark.sql.execution.ui.SparkPlanGraphNode
+
+case class WindowGroupLimitParser(
+    node: SparkPlanGraphNode,
+    checker: PluginTypeChecker,
+    sqlID: Long) extends ExecParser {
+
+  val fullExecName: String = node.name + "Exec"
+
+  override def parse: ExecInfo = {
```

Comment:
Lets add a code comment here explaining when the exec is cosnidered supported.
This will make it easier for us later to understand the logic of looking for "RANK" within the expressions.
Also, it will be good to have a sample of how the node.desc format looks like.
A sample of how it appears in the eventlog will also make it easier to maintain the code and spot in bug when Format changes across spark releases, or when new functionality added to the exec.

Reply:
Added node description and support criteria in method docs